go/cmd/vtctldclient/command/keyspaces.go

```diff
@@ -61,6 +61,24 @@ Otherwise, the keyspace must be empty (have no shards), or returns an error.`,
 		Args:                  cobra.ExactArgs(1),
 		RunE:                  commandDeleteKeyspace,
 	}
+	// DisableVtorcEmergencyReparent disables VTOrc's use of EmergencyReparentShard recoveries on a given keyspace.
+	DisableVtorcEmergencyReparent = &cobra.Command{
```

Comment:
> DisableVtorcEmergencyReparent

Longer alternatives:
1. `DisableKeyspaceVtorcEmergencyReparent`
2. `DisableVtorcKeyspaceEmergencyReparent`

Reply:
On 2nd thought: in a new PR _(or this one)_ I'll probably add support for disabling by shard as well

Keeping "Keyspace" out of the RPC name would make it re-usable for both cases: disabling by shard or keyspace. So I propose sticking with `DisableVtorcEmergencyReparent`

Example - disabling by keyspace:
- `DisableVtorcEmergencyReparent ks1`
-  or `DisableVtorcEmergencyReparent ks1/-` _(not yet supported)_

Example - disabling by shard _(not yet supported)_:
1. `DisableVtorcEmergencyReparent ks1/-80`